<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QWidget" name="LoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1203</width>
    <height>764</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #2c3e50;
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>400</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
background-color: transparent;
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>451</width>
        <height>231</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>10000</width>
        <height>10000</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color: #ecf0f1;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>418</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
background-color: transparent;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="authorise_button">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>150</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: none repeat scroll 0 0 #3498db;
    border: 0 none;
    color: white;
    width: 100%;
    border-radius: 7px;
}

QPushButton:hover {
    border: 2px solid gray;
}</string>
       </property>
       <property name="text">
        <string>Войти</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="login_edit">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>51</y>
         <width>376</width>
         <height>40</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>120</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: none repeat scroll 0 0 #f5f9fa;
    border-radius: 5px;
    margin: 0 auto;
    width: 300px;
border: 1px solid rgba(128, 128, 128, 0.5);

</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Логин:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="password_edit">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>98</y>
         <width>376</width>
         <height>40</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>120</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: none repeat scroll 0 0 #f5f9fa;
    border-radius: 5px;
    margin: 0 auto;
    width: 300px;
border: 1px solid rgba(128, 128, 128, 0.5);</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Пароль:</string>
       </property>
      </widget>
      <widget class="QLabel" name="log_err_msg">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: red;</string>
       </property>
       <property name="text">
        <string>Неверный логин или пароль!</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>81</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>logomiet.png</pixmap>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>login_edit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>LoginForm</receiver>
   <slot>startAnotherAttempt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>656</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>973</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startAnotherAttempt()</slot>
 </slots>
</ui>
